DATA_MODEL_README.md

```diff
@@ -1,6 +1,6 @@
 # Cumulus - Data model
 
-Cumulus uses DynamoDB for database storage. While DynamoDB is a schemaless database storage system, Cumulus enforces schemas on record creation and update using the [`ajv` package](https://github.com/epoberezkin/ajv).
+Cumulus uses RDS Postgres for database storage.
```

Comment:
This probably should have an update of sorts re: how API schemas are enforced.   We're still, essentially, using ajv (for the moment) in addition to the underlying postgres schemas, so this can't be entirely removed.